app/src/test/java/org/mozilla/fenix/HomeActivityTest.kt

```diff
@@ -87,6 +93,42 @@ class HomeActivityTest {
         assertFalse(activity.isActivityColdStarted(startingIntent, null))
     }
 
+    @Test
+    fun `navigateToBrowserOnColdStart in normal mode navigates to browser`() {
+        clearMocks(activity)
```

Comment:
nit: The `clearMocks` call is only needed because of `every { activity.applicationContext } returns testContext` in `@Before`. 

Let's move that one line from `@Before` to the one test that needs it: `getModeFromIntentOrLastKnown returns mode from settings when intent does not set`, then we can remove the `clearMocks` calls in those other tests.